src/main/java/com/conveyal/datatools/editor/models/transit/TripPattern.java

```diff
@@ -47,7 +45,7 @@
 
     public TripDirection patternDirection;
 
-    public List<TripPatternStop> patternStops = new ArrayList<TripPatternStop>();
+    public List<TripPatternStop> patternStops = new ArrayList<>();
 
     // give the UI a little information about the content of this trip pattern
     public transient Long numberOfTrips;
```

Comment:
Why is this a long? Would there ever be more than 2^31 trips on a pattern? I think that would even make the code below crash, since map.size returns an int.